### PR TITLE
Ensure that civicrm_dashboard has a foreign key to civicrm_domain

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
After checking out production db i noticed we only had an index no actual foreign key between civicrm_dashboard and civicrm_domain. This runs an upgrade to make sure there is the FK

Before
----------------------------------------
Maybe index maybe FK

After
----------------------------------------
FK instantiated 

ping @christianwach @eileenmcnaughton

### CODE REVIEW

File: xml/schema/Core/Dashboard.xml [hunk 1]
```diff
@@ -35,6 +35,7 @@
     <table>civicrm_domain</table>
     <key>id</key>
     <add>3.1</add>
+    <onDelete>CASCADE</onDelete>
```
Comment (eileenmcnaughton) [inline xml/schema/Core/Dashboard.xml]: is this right?

**Comment by eileenmcnaughton** [comment]
test fail relates

File: xml/schema/Core/Dashboard.xml (hunk 1 above)
Comment (seamuslee001) [reply xml/schema/Core/Dashboard.xml]: well the column def is a NOT NULL column so its the only one that seemed to make sense to me

Comment (eileenmcnaughton) [reply xml/schema/Core/Dashboard.xml]: ok

**Pull request merged**